db/data/20220713143528_populate_site_settings.rb

```diff
@@ -27,6 +27,7 @@ def up
         provider: 'greenlight' },
       { setting: Setting.find_by(name: 'Terms'), value: '', provider: 'greenlight' },
       { setting: Setting.find_by(name: 'PrivacyPolicy'), value: '', provider: 'greenlight' },
+      { setting: Setting.find_by(name: 'HelpCenter'), value: '', provider: 'greenlight' },
```

Comment:
You'll need to create a new migration for this. Adding it to an existing migration will not work for people who have already run that migration

Reply:
Okay got the new migration files for setting and site_setting. ✅